pgjdbc/src/main/java/org/postgresql/util/LruCache.java

```diff
@@ -55,13 +57,13 @@ protected boolean removeEldestEntry(Map.Entry<Key, Value> eldest) {
         return false;
       }
 
-      Iterator<Map.Entry<Key, Value>> it = entrySet().iterator();
+      Iterator<Map.Entry<Key,Value>> it = entrySet().iterator();
       while (it.hasNext()) {
         if (size() <= maxSizeEntries && currentSize <= maxSizeBytes) {
           return false;
         }
 
-        Map.Entry<Key, Value> entry = it.next();
+        Map.Entry<Key,Value> entry = it.next();
```

Comment:
The above two changes look like just whitespace. I see on line 43, and in other places (as well as my own preference) that space follow commas. So, these two changes should probably be eliminated as well. (I guess these are left over from a prior change to simplify from `Map.Entry<Key, Value>` to `Value`?)

Reply:
I am surprised that checkstyle did not catch that.